### ISSUE TITLE
Error with Test run

### ISSUE THREAD

**Issue by kiranpatil222** [exomiser/Exomiser#497]
Hi,

I just installed latest version on Exomiser 13.2, and ran `example/test-analysis-exome`

```
Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2023-06-07 14:29:26.325 ERROR 618207 --- [           main] o.s.boot.SpringApplication               : Application run failed

java.lang.IllegalStateException: Failed to execute CommandLineRunner
```
have got above error, I have changed the data folder and its version to 2302

**Comment by julesjacobsen** [comment]
Please could you include the command you tried to run and the application.properties along with the rest of the error message.

**Comment by kiranpatil222** [comment]
Hi I ran below command
`java -Xms2g -Xmx4g -jar exomiser-cli-13.2.0.jar --analysis examples/test-analysis-exome.yml`

**Comment by julesjacobsen** [comment]
Ah, this error:

```
CoordinatesOutOfBoundsException: Fully-closed coordinates 1:249149660-249149661 out of contig bounds [1,248956422]
```
Is telling you that the coordinates of a variant in your input VCF 1:249149660-249149661 do not fit within the bounds of the contig [1,248956422] i.e. 249149660 > 248956422

GRCh37 chr1 - 249250621
GRCh38 chr1 - 248956422

This would fit on a GRCh37/hg19 chromosome 1 but not on a GRCh38/hg38 chromosome 1. Are you sure you've told Exomiser the correct assembly your input VCF was called on?

**Comment by kiranpatil222** [comment]
Ohh Ya thanks a lot,, I had changed to hg38 after configuration So was that error, it ran perfectly after i made it to 19
